### PR TITLE
Lionelb/fix elastic mapping

### PR DESCRIPTION
**Pull request by lionelB**
this PR
- add kibana in dev
- optimize mapping for text field

### CODE REVIEW

File: docker-compose.override.prod.yml [hunk 1]
```diff
@@ -37,11 +37,3 @@ services:
       - setup.kibana.host=kibana:5601
       - output.elasticsearch.hosts=elasticsearch:9200
       - output.elasticsearch.enabled=true
-
```
Comment (douglasduteil) [inline docker-compose.override.prod.yml:40]: No Kibana in production ?

**Comment by douglasduteil** [comment]
Why do you need kibana in dev ?
Are you making some cool dashboards with it ?
We might want to share the dev kibana config if yes.

**Comment by lionelB** [comment]
@douglasduteil kibana is needed in dev in order to use the developer console. it's like graphiql, you can write request and see the results.

File: docker-compose.override.prod.yml (hunk 1 above)
Comment (lionelB) [reply docker-compose.override.prod.yml:40]: my bad, I didn't realize I removed it from prod. we just need it in dev too :)

**Comment by douglasduteil** [comment]
You have a kibana config in dev then right ?
With the default index to use at least

**Comment by lionelB** [comment]
Initially, kibana was only use in prod and the ES consultant told us it could be really handy to have it in dev too, So i removed it from `docker-compose.override.prod.yml`and move it to `docker-compose.override.yml`

**Comment by douglasduteil** [comment]
> So i removed it from `docker-compose.override.prod.yml`and move it to `docker-compose.override.yml`

Yes that's cool but, now that you are using it in dev, you should have a valide config for kibana that works in dev and prod (in docker) right ?

**Comment by lionelB** [comment]
why do you think it's not the case ?

**Comment by douglasduteil** [comment]
> why do you think it's not the case ?

I found no proof of it being the case
Can you show it to me ?

**Review by douglasduteil — APPROVED** [review]

**Pull request merged**